File: modul_6/PRAK603-2510817220008-NamiraHumaira.py
n1, n2 = map(int, input().split())
if n1 != n2:
    print("Jumlah tidak sama")
    exit()
list1 = list(map(int, input().split()))
list2 = list(map(int, input().split()))
for i in range(n1):
    print(list1[i] * list2[i], end=" ")

File: modul_6/PRAK603-2510817220008-NamiraHumaira.c
#include <stdio.h>
int main ()
{
    while (1)
    {
        int n1, n2;
        scanf("%d %d", &n1, &n2);
        if (n1 == n2)
        {
            int nilai_1[n1], nilai_2[n2], nilai_hasil[n1]; 
            for (int i = 0; i < n1; i++)
            {
                scanf("%d", &nilai_1[i]);
            }
        
            for (int i = 0; i < n2; i++)
            {
                scanf("%d", &nilai_2[i]);
                }
            printf("\n");
            for (int i = 0; i < n2; i++)
            {
                nilai_hasil[i] = nilai_1[i] * nilai_2[i];
                printf("%d ", nilai_hasil[i]);
            }
            break;
        }
        else
        {
            printf("Jumlah tidak sama\n\n");
            continue;
        }
    }
    return 0;
}